cms/djangoapps/contentstore/management/commands/delete_v1_libraries.py

```diff
@@ -0,0 +1,106 @@
+"""A Command to  Copy or uncopy V1 Content Libraries entires to be stored as v2 content libraries."""
+
+import logging
+from textwrap import dedent
+
+from django.core.management import BaseCommand, CommandError
+
+from opaque_keys.edx.keys import CourseKey
+from opaque_keys.edx.locator import LibraryLocator
+
+from xmodule.modulestore.django import modulestore
+
+from celery import group
+
+from cms.djangoapps.contentstore.tasks import delete_v1_library
+
+from .prompt import query_yes_no
+
+log = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    """
+    Copy or uncopy V1 Content Libraries (default all) entires to be stored as v2 content libraries.
+    First Specify the uuid for the collection to store the content libraries in.
+    Specfiy --all for all libraries, library ids for specific libraries,
+    and -- file followed by the path for a list of libraries from a file.
+
+    Example usage:
+
+        $ ./manage.py cms copy_libraries_from_v1_to_v2 'collection_uuid' --all
```

Comment:
It would be nice to have the correct examples for the delete command, not the copy command